vendor/lowrisc_ibex/dv/verilator/pcount/cpp/ibex_pcounts.cc

```diff
@@ -53,7 +53,7 @@ static bool has_hpm_counter(int index) {
 
 std::string ibex_pcount_string(bool csv) {
   char separator = csv ? ',' : ':';
-  std::string::size_type longest_name_length;
+  std::string::size_type longest_name_length = 0;
```

Comment:
Interesting that this wasn't flagged before. What behavior did this cause?

Reply:
When trying to build the verilator simulation using:

```
fusesoc --cores-root=. run --target=sim --tool=verilator --setup --build lowrisc:ibex:demo_system
```

I got the following error:

```
ERROR: ../src/lowrisc_dv_verilator_ibex_pcounts_0/cpp/ibex_pcounts.cc: In function ‘std::string ibex_pcount_string(bool)’:
../src/lowrisc_dv_verilator_ibex_pcounts_0/cpp/ibex_pcounts.cc:80:41: warning: ‘longest_name_length’ may be used uninitialized [-Wmaybe-uninitialized]
   80 |       int padding = longest_name_length - ibex_counter_names[i].length();
      |                     ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/lowrisc_dv_verilator_ibex_pcounts_0/cpp/ibex_pcounts.cc:56:26: note: ‘longest_name_length’ was declared here
   56 |   std::string::size_type longest_name_length;
      |                          ^~~~~~~~~~~~~~~~~~~
```

The full error message when building with the wrong verilator version (5.020) as well is in issue #125

Reply:
This is a change to vendored code, which essentially lives in another repository. The original file lives here: https://github.com/lowRISC/ibex/blob/master/dv/verilator/pcount/cpp/ibex_pcounts.cc
Would you like to make a PR in the Ibex repo or shall I do that for you?

Reply:
I've made a PR to fix this: https://github.com/lowRISC/ibex/pull/2227